css/styles.css

```diff
@@ -215,6 +224,193 @@ body {
   margin-top: -40px;
 }
 
+/************** WORKS SECTION ***************/
+.works-sec {
+  display: grid;
+  grid-template-columns: 20% 20% 20% 20% 20%;
+}
+
+.project {
+  margin-top: 4%;
+  margin-bottom: 4%;
+  grid-column: 2/6;
+  width: 70%;
+  background-color: #fff;
+  border-radius: 1%;
+}
+
+.img-prg {
+  width: 40vh;
+  height: 40vh;
+  margin-top: 20px;
+  margin-left: 20px;
+  margin-bottom: 20px;
+}
+
+.project-info {
+  width: 50%;
+  margin: auto;
+}
+
+.project-title {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 32px;
+  width: 60%;
+}
+
+.project-det {
+  font-size: 13px;
+  justify-content: space-between;
+  width: 50%;
+}
+
+.project-det-desc,
+.project-det-yr {
+  color: #7a869a;
+}
+
+.prgt-det-circl {
+  background-image: url('../assets/prgt-circle-det.svg');
+  margin-top: 1%;
+  background-repeat: no-repeat;
+  width: 10px;
+}
+
+.project-desc {
+  font-size: 15px;
+  color: #344563;
+}
+
+.project-tags ul {
+  margin: 0;
+  list-style-type: none;
+  justify-content: space-between;
+  width: 30%;
+}
+
+.project-tag {
+  list-style: none;
+  background-color: #ebebff;
+  color: #6070ff;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 2%;
+  border-radius: 10%;
+}
+
+.project-link {
+  margin-top: 2%;
+  margin-bottom: 4%;
+  background-color: transparent;
+  padding: 2%;
+  font-size: 17px;
+  border-radius: 4%;
+  color: #396df2;
+  border-color: #396df2;
+}
+
+/************** ABOUT SECTION ***************/
+
+.about-me {
+  background-color: #fff;
+  border-top-right-radius: 200px;
+}
+
+.about-container {
+  width: 60%;
+  margin: auto;
+  padding: 4%;
+}
+
+.about-section-one {
+  width: 40%;
+}
+
+.languages {
+  width: 50%;
+  padding: 4%;
+}
+
+.socials-about {
+  list-style-type: none;
+  justify-content: space-between;
+  width: 40%;
+  margin-left: -35px;
+}
+
+.about-title {
+  color: #091e42;
+  font-size: 40px;
+  width: 50%;
+}
+
+.about-desc {
+  color: #091e42;
+  font-size: 16px;
+}
+
+.languages-content {
+  display: none;
+}
+
+.collapse-btn {
+  padding: 5%;
+  margin-top: 4px;
+  border-radius: 0;
+  border-top: 0;
+  border-left: 0;
+  border-right: 0;
+  border-color: #dfe1e6;
+  background-color: #fff;
+  font-size: 20px;
+  font-weight: 500;
+  text-align: left;
+}
+
+.collapse-btn::after {
+  content: '\003E';
+  color: #acb7c3;
+  float: right;
+}
+
+.active {
+  border-bottom: 0;
+}
+
+.active::after {
+  content: '\003E';
+  color: #acb7c3;
+  -webkit-transform: rotate(90deg);
+  transform: rotate(90deg);
+}
```

Comment:
- [ ] In addition to this active effect, It is a requirement to give all your buttons a hover effect so that a scrolling user can easily tell they are clickable elements. e.g.
```
button: hover span {
  cursor: pointer;
}
```
-  N/B: Don't worry about any javascript functionality.

- [ ] Give all your buttons a border radius so that they can match as displayed in the Figma design template you chose

Reply:
thank you for the feedback.. I was able to implement the feedback.. have a look and let me know.